osquery/tables/utility/get_file.cpp

```diff
@@ -0,0 +1,166 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <sys/stat.h>
+
+#include <boost/filesystem.hpp>
+
+#include <osquery/filesystem.h>
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+namespace fs = boost::filesystem;
+
+namespace osquery {
+namespace tables {
+
+const std::map<fs::file_type, std::string> kTypeNames{
```

Comment:
Maybe put this into `./include/filesystem.h`:
```cpp
extern const std::map<fs::file_type, std::string> kTypeNames;
```

 and the definition in `./osquery/filesystem/filesystem.cpp`, don't duplicate: https://github.com/facebook/osquery/blob/master/osquery/tables/utility/file.cpp#L24